### ISSUE TITLE
Forge version

### ISSUE THREAD

**Issue by EGOIST1372** [Julienraptor01/Eating-Animation-Smaller-Utilities-Compatibility#1]
someone asked for forge version since already existing pack doesn't work for them. I link them to here so they can explain better their problem

**Comment by Julienraptor01** [comment]
Yes my pack won't work on the forge port as it uses a different file organization.
I won't support the forge port of the mod if they don't change to the original mod file organization.
This is simply because i use the original fabric mod and i have limited resources in term of time.
But if at some moment the original mod change the file organization to match the forge port then my pack will support it as i'll follow the original fabric mod.
Feel free to use my work to do it yourself as long as you follow the license.
